Client/Frontend/Browser/URIFixup.swift

```diff
@@ -3,12 +3,17 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import Foundation
+import UIKit
 
 class URIFixup {
     static func getURL(entry: String) -> NSURL? {
         let trimmed = entry.stringByTrimmingCharactersInSet(NSCharacterSet.whitespaceAndNewlineCharacterSet())
         var url = NSURL(string: trimmed)
 
+        if !isURL(trimmed) && !trimmed.contains("localhost") {
```

Comment:
Nit: This would be better using a `guard` since we're checking preconditions that the string is a URL and not localhost.